package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "calaculator-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "big.js": "^6.1.1",
+    "react": "^17.0.2",
```

Comment:
- [ ] It would be better if we add the `prop-types` package to the project. This will help us to type check our props